test/expr/cast/set_coerce.swift

```diff
@@ -18,4 +18,4 @@ var setD = Set<D>()
 
 // Test set upcasts
 setC = setD
-setD = setC // expected-error{{cannot assign value of type 'Set<C>' to type 'Set<D>'}}
+setD = setC // expected-error{{cannot convert value of type 'Set<C>' to 'Set<D>' in assignment, arguments to generic parameter 'Element' ('C' and 'D') are expected to be equal}}
```

Comment:
Actually it seems like for assignment it make make sense to go with original text "cannot assign value of type ... to type ...`

Reply:
Yeah, on second thought, that does look much nicer. I'll make that change in my next round of revisions 👍